src/main.cpp

```diff
@@ -2809,6 +2809,7 @@ bool AcceptBlockHeader(const CBlockHeader& block, CValidationState& state, CBloc
     uint256 hash = block.GetHash();
     BlockMap::iterator miSelf = mapBlockIndex.find(hash);
     CBlockIndex *pindex = NULL;
+    if (hash != chainparams.GetConsensus().hashGenesisBlock) {
     if (miSelf != mapBlockIndex.end()) {
```

Comment:
I'd prefer those merge conflicts to stay merge conflicts, as it forces you to understand if that additional check matters for your merge.

What can I say, I once fixed a bug where a lack of indentation of a newly added bit of code lead to a mistake that could have potentially played a part of getting someone killed, so... :/